packages/medusa/src/api/routes/admin/regions/create-region.ts

```diff
@@ -46,6 +54,9 @@ import { EntityManager } from "typeorm"
  *             type: array
  *             items:
  *               type: string
+ *         includes_tax:
+ *             description: "[EXPERIMENTAL] Is the region tax inclusive or not"
```

Comment:
suggestion: Rephrase to improve semantics
```suggestion
 *             description: "[EXPERIMENTAL] Tax included in prices in region"
```